src/site/content/en/blog/20things-pageflip/index.md

```diff
@@ -0,0 +1,419 @@
+---
+layout: post
+title: Case Study - Page Flip Effect from 20thingsilearned.com
+authors:
+  - hakimelhattab
+date: 2011-01-20
+tags:
+  - blog
+  - case-study
+---
+## Introduction
+
+In 2010, [F-i.com](http://www.f-i.com) and the
+Google Chrome team collaborated on an HTML5-based educational web app called
+20 Things I Learned about Browsers and the Web
+([www.20thingsilearned.com](http://www.20thingsilearned.com)).
+One of the key ideas behind this project was that it would best be presented
+in the [context of a book](http://chrome.blogspot.com/2010/11/curious-guide-to-browsers-and-web.html).
+Since the contents of the book is very much about open web
+technologies we felt it was important to stay true to that by making the
+container itself an example of what these technologies allow us to
+accomplish today.
+
+<figure>
+{% Img src="image/T4FyVKpzu4WKF1kBNvXepbi08t52/o5841WYMvQebTxtGDvU3.png", alt="Book cover and homepage of '20 Things I Learned About Browsers and the Web'", width="500", height="347" %}
+<figcaption>Book cover and homepage of "20 Things I Learned About Browsers and the Web" (<a href="http://www.20thingsilearned.com" target="_blank">www.20thingsilearned.com</a>)</figcaption>
+</figure>
+
+We decided that the best way to achieve the feeling of a real world
+book is to simulate the good parts of the analogue reading experience
+while still leveraging the benefits of the digital realm in areas such
+as navigation. A lot of effort went into the graphical and interactive
+treatment of the reading flow - especially how the pages of the books
+flip from one page to another.
+
+## Getting Started
+
+This tutorial will take you through the process of creating your own
+page flip effect using the canvas element and plenty of JavaScript. Some
+of the rudimentary code, such as variable declarations and event listener
+subscription, has been left out of the snippets in this article so remember
+to reference the working example.
+
+Before we get started it's a good idea to [check out the demo](https://www.html5rocks.com/static/demos/20things_pageflip/example/index.html) so that you know what we`re aiming to build.
```

Comment:
```suggestion
Before we get started it's a good idea to [check out the demo](https://www.html5rocks.com/static/demos/20things_pageflip/example/index.html) so that you know what we're aiming to build.
```